src/components/main/friends/friend/mod.rs

```diff
@@ -92,12 +109,35 @@ pub fn Friend<'a>(cx: Scope<'a, Props>) -> Element<'a> {
                             icon: Shape::XMark,
                             state: ui_kit::button::State::Danger,
                             on_pressed: move |_| {
-                                let mut multipass = cx.props.account.clone();
-                                let did_to_remove = cx.props.friend.clone();
-                                match multipass.remove_friend(&did_to_remove) {
-                                    Ok(_) => {}
-                                    Err(_) => {
-                                        log::debug!("error removing friend");
+                                let local_state = use_atom_ref(&cx, STATE).clone();
```

Comment:
this will break the UI. the same hooks must be created across all invocations of an Element. creating a hook in a callback, loop, or conditional breaks that rule.

Reply:
Resolved by the latest commit